src/op_mode/openvpn.py

```diff
@@ -142,6 +144,17 @@ def _get_interface_status(mode: str, interface: str) -> dict:
 
     return data
 
+def _get_interface_state(iface):
+    rc, out = rc_cmd(f'ip --json link show dev {iface}')
+    data = json.loads(out)
+    return data[0].get('operstate', 'DOWN')
+
```

Comment:
jestabro# set interfaces openvpn vtun2 disable
[edit]
jestabro# commit
[edit]
jestabro# ip --json link show dev vtun2
Device "vtun2" does not exist.

A (hacky) fix is:
```
def _get_interface_state(iface):
    rc, out = rc_cmd(f'ip --json link show dev {iface}')
    # in case out == 'Device ... does not exist.'
    try:
        data = json.loads(out)
    except:
        return 'DOWN'
    return data[0].get('operstate', 'DOWN')
```
and similar below for _get_interface_description.

The problem appears with the fix for T5149.

Reply:
@jestabro Thanks, Done!